vault/core.go

```diff
@@ -1230,6 +1234,7 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 			return nil, fmt.Errorf("core setup failed, could not verify plugin directory: %w", err)
 		}
 	}
+	c.pluginTmpdir = conf.PluginTmpdir
```

Comment:
Do we need to support relative paths for PluginTempdir like PluginDirectory does above?

Reply:
Good point, added 👍